src/Styleguide/Utils/BreakpointVisualizer.tsx

```diff
@@ -0,0 +1,89 @@
+import { Display } from "@artsy/palette"
+import { isEqual } from "lodash"
+import React from "react"
+import Transition from "react-transition-group/Transition"
+import styled from "styled-components"
+import { Box } from "Styleguide/Elements/Box"
+import { Responsive } from "Utils/Responsive"
+
+const StyledBox = styled(Box)`
+  position: fixed;
+  background-color: black;
+  color: white;
+  border-bottom-right-radius: 5px;
+  transition: opacity 0.3s ease-in-out;
+  opacity: 0;
+`
+
+const BreakpointText = ({ xs, sm, md, lg }) => {
+  return (
+    <Display size="8">
+      {xs ? "xs" : sm ? "sm" : md ? "md" : lg ? "lg" : "xl"}
+    </Display>
+  )
+}
+
+interface BreakpointVisualizerState {
+  breakpoints: any
+  changed: boolean
+}
+
+export class BreakpointVisualizer extends React.Component<
+  {},
+  BreakpointVisualizerState
+> {
+  constructor(props) {
+    super(props)
+    this.state = {
+      breakpoints: {},
+      changed: false,
+    }
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (!isEqual(prevState.breakpoints, this.state.breakpoints)) {
+      this.setState({
+        changed: true,
+      })
+    }
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        {!navigator.userAgent.match(/Chromatic/) && (
```

Comment:
what is `navigator` here? Is that some global or something?

Reply:
Yep. It's generally on the window... might need to check for its existence I guess.